SmartDeviceLink_Example/Classes/ProxyManager.m

```diff
@@ -324,7 +333,7 @@ - (void)prepareRemoteSystem {
         dispatch_group_leave(dataDispatchGroup);
 
         if (success == NO) {
-            NSLog(@"Something went wrong, image could not upload: %@", error);
+            SDLLogW(@"Something went wrong, image could not upload: %@", error);
```

Comment:
Ehhh...it's right on that edge. I'd think of an error as something more serious, while a warning is an error that doesn't materially affect progress. An image can fail to upload and future things will still be fine.